c/driver/postgresql/CMakeLists.txt

```diff
@@ -53,14 +42,15 @@ add_arrow_lib(adbc_driver_postgresql
               ${ADBC_LINK_FLAGS}
               SHARED_LINK_LIBS
               ${LIBPQ_LINK_LIBRARIES}
-              nanoarrow
               STATIC_LINK_LIBS
               ${LIBPQ_LINK_LIBRARIES}
               nanoarrow
               ${LIBPQ_STATIC_LIBRARIES})
 include_directories(SYSTEM ${REPOSITORY_ROOT})
 include_directories(SYSTEM ${REPOSITORY_ROOT}/c/)
 include_directories(SYSTEM ${LIBPQ_INCLUDE_DIRS})
+include_directories(SYSTEM ${REPOSITORY_ROOT}/c/vendor)
```

Comment:
Ideally we should use target_include_directories now that we have multiple targets

Reply:
Gave this a shot but I wasn't able to get it to play nicely with `add_arrow_lib` - seems like it was just being ignored? Happy to look deeper in a follow up

Reply:
I think there were upstream changes to Arrow's CMake to enable that to work after we forked from there, now that I think about it. We can leave that for a follow up.

Reply:
Makes sense. I'll take a look upstream next